src/ui/components/NetworkMonitor/index.tsx

```diff
@@ -32,14 +32,16 @@ export const NetworkMonitor = ({
   selectFrame,
 }: PropsFromRedux) => {
   const [selectedRequest, setSelectedRequest] = useState<RequestSummary>();
-  const [types, setTypes] = useState<Set<RequestType>>(new Set(["xhr", "javascript", "html"]));
+  const [types, setTypes] = useState<Set<CanonicalRequestType>>(
+    new Set(["Fetch/XHR", "Javascript", "HTML"])
+  );
   const [vert, setVert] = useState<boolean>(false);
 
   const container = useRef<HTMLDivElement>(null);
 
   const closePanel = () => setSelectedRequest(undefined);
 
-  const toggleType = (type: RequestType) => {
+  const toggleType = (type: CanonicalRequestType) => {
     if (types.has(type)) {
       types.delete(type);
```

Comment:
Ohh just spotted this, also a bit questionable to mutate the existing set. Better to do
```
const toggleType = (type: CanonicalRequestType) => {
  const newTypes = new Set(types);
  if (newTypes.has(type)) {
    newTypes.delete(type);
  } else {
    newTypes.add(type);
  }
  setTypes(newType);
};
```